gateway-service/routes/data_retrieval.py

```diff
@@ -28,20 +28,37 @@ def createWeatherData():
     try:
         session_service_response = check_user_session(request.headers.get('Access-Token'))
         request_body = json.loads(request.data)
+        type = request_body.get('type')
         user_id = session_service_response.get('user_id')
-        kafka_producer.send('data-retrieval-queue', json.dumps({
-            "user_id": user_id,
-            "date": request_body.get('date'),
-            "time": request_body.get('time'),
-            "station":request_body.get('station')
-        }, default=json_util.default).encode('utf-8'))
+        if type == 'nexrad':
         
-        kafka_producer.send('audit-queue', json.dumps({
-            "user_id": user_id,
-            "response": {},
-            "request": request.data,
-            "service_provider_identifier": 'data-retrieval-service'
-        }, default=json_util.default).encode('utf-8'))   
+            kafka_producer.send('data-retrieval-queue', json.dumps({
```

Comment:
just assign JSON inside to variable
example :

if type == 'nexrad':
     j1 = json.dumps({"acv": 'x1'})
elif type == 'merra':
   j1 = json.dumps({"wfwe": "few"})

kafka_producer.send('queue_name', j1)
kafka_producer.send('queue_namne2', j2)